### DISCUSSION TITLE
Not found page (404) with routing (Next pages router)

### DISCUSSION THREAD

**Discussion by huytv04883** [amannn/next-intl#1507] [Q&A]
### Description

How to define 404 page in [locale] folder? with pages router

### Verifications

- [X] I've verified that the problem I'm experiencing isn't covered in [the docs](https://next-intl-docs.vercel.app/).
- [X] I've searched for similar, existing issues on [GitHub](https://github.com/amannn/next-intl/issues) and [Stack Overflow](https://stackoverflow.com/search?q=next-intl).
- [X] I've compared my app to [a working example](https://next-intl-docs.vercel.app/examples) to look for differences.

### Mandatory reproduction URL

Pages router

### Reproduction description

Steps to reproduce:
1. Open reproduction
2. Click on …
3. See error: …

### Expected behaviour

How to define 404 page in [locale] folder? with pages router

**Comment by amannn** [comment]
Moving this to a discussion since it's a usage question …

**Comment by emirxmertoglu** [comment]
Take a look, https://next-intl-docs.vercel.app/docs/environments/error-files
